config/environments/development.rb

```diff
@@ -51,4 +51,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
```

Comment:
Should you also add this line to the test environment config? I don't know much about this, but I don't know what would happen now if we tested email sending

Reply:
Ok